docs/zh/latest/plugins/zipkin.md

```diff
@@ -27,250 +27,239 @@ description: 本文介绍了关于 Apache APISIX zipkin 插件的基本信息及
+该插件还支持将追踪数据发送到其他兼容的收集器，例如 [Jaeger](https://www.jaegertracing.io/docs/1.51/getting-started/#migrating-from-zipkin) 和 [Apache SkyWalking](https://skywalking.apache.org/docs/main/latest/en/setup/backend/zipkin-trace/#zipkin-receiver)，这两者都支持 Zipkin [v1](https://zipkin.io/zipkin-api/zipkin-api.yaml) 和 [v2](https://zipkin.io/zipkin-api/zipkin2-api.yaml) API。
 
-## 属性
+## 静态配置
 
-| 名称         | 类型    | 必选项 | 默认值        | 有效值       | 描述                                                                 |
-| ------------ | ------ | ------ | ------------ | ------------ | -------------------------------------------------------------------- |
-| endpoint     | string | 是     |              |              | Zipkin 的 HTTP 节点。例如：`http://127.0.0.1:9411/api/v2/spans`。      |
-| sample_ratio | number | 是     |              | [0.00001, 1] | 对请求进行采样的比例。当设置为 `1` 时，将对所有请求进行采样。              |
-| service_name | string | 否     | "APISIX"     |              | 需要在 Zipkin 中显示的服务名称。                                        |
-| server_addr  | string | 否     | $server_addr |              | 当前 APISIX 实例的 IPv4 地址。                                        |
-| span_version | integer| 否     | 2            | [1, 2]       | span 类型的版本。                                                      |
+默认情况下，`zipkin` 插件的 NGINX 变量配置在 [默认配置](https://github.com/apache/apisix/blob/master/apisix/cli/config.lua) 中设置为 `false`：
 
-在当前版本中，每个被跟踪的请求都会创建如下所示的 span：
+要修改此值，请将更新后的配置添加到 `config.yaml` 中。例如：
 
+```yaml
+plugin_attr:
+  zipkin:
+    set_ngx_var: true
 ```
-request
-├── proxy: from the beginning of the request to the beginning of header filter
-└── response: from the beginning of header filter to the beginning of log
-```
-
-在旧版本（将 `span_version` 属性设置为 `1`）中，将创建如下 span：
-
-```
-request
-├── rewrite
-├── access
-└── proxy
-    └── body_filter
-```
-
-:::note
 
-上述 span 的名称与同名的 NGINX phase 没有联系。
+重新加载 APISIX 以使更改生效。
 
-:::
+## 属性
 
-### 上游服务示例
+查看配置文件以获取所有插件可用的配置选项。
 
-```go title="Go with Gin"
-func GetTracer(serviceName string, port int, enpoitUrl string, rate float64) *zipkin.Tracer {
-    // create a reporter to be used by the tracer
-    reporter := httpreporter.NewReporter(enpoitUrl)
-    // set-up the local endpoint for our service host is  ip:host
+| 名称          | 类型     | 是否必需    | 默认值        | 有效值      |      描述     |
+|--------------|---------|----------|----------------|-------------|------------------|
+| `string`     | 字符串   | 是       |                |            | 要 POST 的 Zipkin span 端点，例如 `http://127.0.0.1:9411/api/v2/spans`。 |
+|`sample_ratio`| 数字    | 是       |                | [0.00001, 1] | 请求采样频率。设置为 `1` 表示对每个请求进行采样。    |
+|`service_name`| 字符串  | 否       | "APISIX"       |              | 在 Zipkin 中显示的服务名称。   |
+|`server_addr` | 字符串  | 否       | `$server_addr` 的值 | IPv4 地址 | Zipkin 报告器的 IPv4 地址。例如，可以将其设置为你的外部 IP 地址。 |
+|`span_version`| 整数    | 否       | `2`            | [1, 2]       | span 类型的版本。    |
```

Comment:
<img width="282" alt="image" src="https://github.com/user-attachments/assets/0e7a310d-6340-44d6-a07d-cf15805d3ec0" />

this col was not translated in other docs